R/stabsel.R

```diff
@@ -20,6 +20,15 @@ stabsel.matrix <- function(x, y, fitfun = lars.lasso, args.fitfun = list(),
 
     cll <- match.call()
     p <- ncol(x) ## TODO: what about intercept?
+    if (missing(y)) {
```

Comment:
Please do not base this ONLY on a missing `y` but also on the class of your fitter